### PR TITLE
Add string split function

### PR DESCRIPTION
**Pull request by CsokiHUN**

### CODE REVIEW

File: src/api/player/server/getPlayerIdentifiers.lua [hunk 1]
```diff
@@ -3,21 +3,11 @@
...
-    return (identifiers)
-end
\ No newline at end of file
+	local identifiers = {}
+	for _, v in pairs(GetPlayerIdentifiers(playerSrc)) do
+		local splitted = _NCS.Strings:split(v, ":")
+
+		identifiers[splitted[1]] = v
+	end
+	return identifiers
```
Comment (PABLO-1610) [inline src/api/player/server/getPlayerIdentifiers.lua:12]:
```suggestion
	return (identifiers)
```

**Review by PABLO-1610 — CHANGES_REQUESTED** [review]
Hi, good first PR 👍 

Juste some little adjustements.

Thanks a lot for contributing, don't hesitate to join our Discord to redeem your contributor role.

**Review by CsokiHUN — COMMENTED** [review]
Next time I will try to follow the schema and use Lua 5.4.

**Review by PABLO-1610 — APPROVED** [review]
Thanks a lot and welcome into the project 😁

**Pull request merged**